projects/storefrontstyles/.npmignore

```diff
@@ -2,3 +2,6 @@ yarn.lock
 importer.js
 temp-scss/
 .release-it.*
+.pipeline/
+sub-folder/
+vault*
```

Comment:
I'm assuming vault is where we store the secrets in hyperspace. This is only generated under the hood when running the pipeline right as I can't find that folder in the private repo

Reply:
Hmm not sure why I put it there, I think I saw files related to that path but I don't see anything after removing it from the npmignore. I'll remove it :)

Reply:
now I have to ask about these 

```
staging_output_deprecated.json
stage-config.json
errorDetails.json
```

Reply:
not sure, originally saw those files in the artifacts, thats why i included them. but i dont see them anymore... removed them

Reply:
consensus that we agreed upon is to delete it from source, and publish it automatically during the sync with the proper branch value